src/OrbitGl/App.cpp

```diff
@@ -1796,11 +1796,15 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadSymbols(
     return it->second;
   }
 
+  ModuleData* module_data = GetMutableModuleByPathAndBuildId(module_file_path, module_build_id);
```

Comment:
Just quickly looking over this out of curiosity. This can probably be a const pointer obtained via `GetModuleByPathAndBuildId` (without mutable)

Reply:
Yes, absolutely. Will fix.